### PR TITLE
DMP-1455 MFA B2C Changes 

### PR DESCRIPTION
**Pull request by DavidTMann**
DMP-1455

MFA B2C changes, formatting for text, and new account details

### CODE REVIEW

File: server/assets/javascripts/mfa.js [hunk 1]
```diff
@@ -29,6 +29,51 @@ function insertTitle(){
...
+    </div>`
+
+    $(accountDetailsHtml).insertAfter('#attributeList');
+}
+
+function getAccountDetails(){
+    const authDetails = $('#totpQrCodeControl-text').text();
+
+    const details = {};
+    const externalIdMatch = authDetails.match(/hmcts-stg-external-id:([^?]+)/);
```
Comment (davet1985) [inline server/assets/javascripts/mfa.js:57]: You can't rely on this text as it will be different on each environment

Comment (DavidTMann) [reply server/assets/javascripts/mfa.js]: Updated regex now relies on 2nd : until 1st ?

File: server/assets/javascripts/mfa.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 
 function insertBackLink(){
-    $(`<a href="${document.referrer}" class="govuk-back-link">Back</a>`).insertBefore("#attributeVerification");
+    $('button#cancel').insertBefore("#attributeVerification");
+    // $(`<a href="${document.referrer}" class="govuk-back-link">Back</a>`).insertBefore("#attributeVerification");
```
Comment (davet1985) [inline server/assets/javascripts/mfa.js:4]: Remove comment? or will you tidy this up later?

File: server/assets/javascripts/mfa.js (hunk 1 above)
Comment (davet1985) [reply server/assets/javascripts/mfa.js]: 🤞

**Review by davet1985 — APPROVED** [review]

**Pull request merged**